src/sidebar/services/test/search-filter-test.js

```diff
@@ -57,6 +57,21 @@ describe('sidebar.search-filter', () => {
       assert.equal(result.any[2], 'hi-fi');
       assert.equal(result.any[3], 'a:bc');
     });
+
+    it('supports quoting terms', () => {
+      const parsed = searchFilter.toObject('user:"Dan Whaley"');
+      assert.deepEqual(parsed, {
+        user: ['Dan Whaley'],
```

Comment:
Not that I think Dan cares but do we want to have his name in the code?

Reply:
I used this here because this was literally the example we were using in the Slack discussion that prompted me to write these tests. I think I can get away with it ;)

Reply:
May it be forever in the git history! lol